mcts4py/PuctWithDPWSolver.py

```diff
@@ -26,9 +26,12 @@ def __init__(self,
                  early_stop_condition: dict = None,
                  exploration_constant_decay=1,
                  dpw_exploration=1,
-                 dpw_alpha=1):
-        self.probabilities = pd.read_csv('lr_models/probabilities.csv', index_col=0)
-        self.probabilities.set_index(['port', 'refuel_amount'], inplace=True)
+                 dpw_alpha=1,
+                 probabilities=None):
+        
+        self.probabilities = probabilities
+        # self.probabilities = pd.read_csv('lr_models/probabilities.csv', index_col=0)
```

Comment:
@altineel do not put the `lr_models/probabilities.csv` file into the mcts4py lib. Add them as extra arguments from the domain side code.